src/format.ts

```diff
@@ -37,12 +37,21 @@ export interface PostmanCollection {
     schema: string;
   };
   item: PostmanFolder[];
+  auth?: {
+    type: string;
+    apikey: {
+      key: string;
+      value: string;
+      type: string;
+    }[];
+  };
 }
 
 function queryToItem(
   query: FormattedQuery,
   url: string,
   headers?: Record<string, string>,
```

Comment:
We should change this type to `Array<[string,string]>`, and make it mandatory (it will just be empty if no headers).